### PR TITLE
Clean optimize

### PR DESCRIPTION
**Pull request by crazypsycho**
- Fix issues
- Improve performance
- Remove unused things
- Add jQuery-Migrate-3.0.1

### CODE REVIEW

File: src/Lib/Elebee.php [hunk 1]
```diff
@@ -358,27 +291,33 @@ private function defineAdminHooks() {
...
         $this->loader->addAction( 'init', Config::class, 'cleanUpHead' );
-        $this->loader->addAction( 'init', Config::class, 'disableEmojies' );
         $this->loader->addFilter( 'status_header', Config::class, 'disableRedirectGuess' );
 
         $elebeePublic = new ElebeePublic( $this->getThemeName(), $this->getVersion() );
 
         $this->loader->addAction( 'wp_head', $elebeePublic, 'embedGoogleAnalytics', 0 );
         $this->loader->addAction( 'wp_head', $elebeePublic, 'embedIEConditionals', 0 );
-        $this->loader->addAction( 'wp_enqueue_styles', $elebeePublic, 'enqueueStyles' );
+        $this->loader->addAction( 'wp_enqueue_scripts', $elebeePublic, 'enqueueStyles' );
```
Comment (ghost) [inline src/Lib/Elebee.php:305]: okay we are enqueueStyles as wp_enqueue_scripts?

Comment (crazypsycho) [reply src/Lib/Elebee.php:305]: Yes, because there is no hook explicitly for styles.

File: src/Public/partials/ie-conditionals.php [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <!--[if lt IE 9]>
```
Comment (ghost) [inline src/Public/partials/ie-conditionals.php:1]: Only Lesser then IE 9? 
What is with the style Changes for IE 9 They get removed but not added here

Comment (ghost) [reply src/Public/partials/ie-conditionals.php:1]: My Bad dam github UI doesn't show complete File

**Review by ghost — APPROVED** [review]
No Problem with the Merging of these Changes

**Pull request merged**